mint/run/core/s3select/utils.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage,
+# (C) 2015-2020 MinIO, Inc.
```

Comment:
no need to add 2015, its not added in 2015 so should just say 2020

Reply:
> no need to add 2015, its not added in 2015 so should just say 2020

This is not a new code, just moved from tests.py, shall I still only put 2020 ?

Reply:
in that case you decide what you want, all new code should say 2020